packages/node_modules/@webex/internal-plugin-wdm/src/device/device.js

```diff
@@ -141,9 +141,8 @@ const Device = WebexPlugin.extend({
     // Fun Fact: setTimeout returns a Timer object instead of a Number in Node 6
     // or later
     logoutTimer: 'any',
-    inNetworkLogoutTimer: 'any', // Second logout timer when client is connected to org's internal network
     lastUserActivityDate: 'number',
-    lastInNetworkUserActivityDate: 'number', // Update last user activity date to reset inNetworkInactivityTimer
+    isReachabilityChecked: 'boolean', // To prevent url reachabilty from being called more than once on successful check.
```

Comment:
Since the timer is only expected to run once when the web client loads up the first time i'd say the intranet inactivity check would only run once. If the client goes off/on network unless the browser is refreshed it wouldn't know of the change in network.

Reply:
I don't think this is the intended behavior but I'm not the PM or Eng on it. That's up to you two to sort out